src/test/java/IvanFFavoriteTests.java

```diff
@@ -0,0 +1,109 @@
+import PagesIvaF.AllSongsIvan;
+import PagesIvaF.AllSongsIvan;
+import PagesIvaF.HomePageIvan;
+import PagesIvaF.LoginPageIvan;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.testng.annotations.TestInstance;
+import java.time.Duration;
+
+
+public class IvanFFavoriteTests extends BaseTestIvan{
+    @Test
+    public void checkFavorite() throws InterruptedException{
+        LoginPageIvan loginPage = new LoginPageIvan(getDriver());
+        HomePageIvan homePageIvan = loginPage.login();
```

Comment:
**Please, remove code redundancies!**

**Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )!**

For example

```
 @BeforeMethod
  @Test
  public void LogIn() {
    LoginPageIvan loginPage = new LoginPageIvan(getDriver());
    HomePageIvan homePageIvan = loginPage.login();

  }

  @Test
  public void checkFavorite() throws InterruptedException {
    HomePageIvan homePageIvan = new HomePageIvan(getDriver());
    homePageIvan.favorite();
    System.out.println("Number of favorites         " + (homePageIvan.getNumberfavourite()));

  }
```

or

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >

<suite name="TestNGIvan" parallel="methods" thread-count="2">
    <parameter name="BaseUrl" value="https://bbb.testpro.io/"/>
    <test name="Example Test" preserve-order="false">
        <classes>
          <class name="IvanFFavoriteTests"/>
          <class name="TestLogin">
                <methods>
                <include name="LoginWithValidEmailPasswordTest" />
                </methods>
          </class>
        </classes>
    </test>
</suite>
```